docs/authorization-configuration.md

```diff
@@ -71,4 +73,14 @@ auth: {
 - **redirect_uri** contains the URL to return to after login. The default application root is `/`.
 - **response_type** defaults to the **id_token**. Any other parameter that is added to oAuthData is also added to the authorization payload.
 - **nonceFn** provides a function that returns a string in order to override the default **nonce**.
-- **logoutFn** provides a function to override **logoutUrl** functionality for a custom logout. It needs to execute the function **logoutCallback()** after logout.
\ No newline at end of file
+- **logoutFn** provides a function to override **logoutUrl** functionality for a custom logout. It needs to execute the function **logoutCallback()** after logout.
+
+### Third-party cookies and silent token refresh
+
+OpenID connect configuration lets you specify **automaticSilentRenew** option. When it is set to `true` Luigi will try to automatically renew the token in the background before it expires. Be aware that this mechanism requires the browser to support [third-party cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).
+It is possible to detect whether user's browser supports it by using the script in [`third-party-cookies`](https://github.com/kyma-project/luigi/tree/master/core/third-party-cookies) catalog. Deploy these files on a **different domain** than your main application and set `thirdPartyCookiesScriptLocation` to the `init.html` file. Luigi will then detect the cookies support at the initialization and warn in the console if user's browser has it disabled.
+
+In addition, when Luigi fails to renew the token and then logs out the user, it will add query params `?reason=tokenExpired&thirdPartyCookies=[VALUE]`, where `VALUE` is one of (`disabled`, `enabled`, `not_checked`) to the logout page redirect. Application developer can read these params and provide proper logout page based on them. 
```

Comment:
``` 
When Luigi fails to renew the token and then logs out the user, it adds the `?reason=tokenExpired&thirdPartyCookies=[VALUE]` query parameters . You can substitute the  **VALUE** with one of the following: 

```
We can then introduce the three values the user can enter here.

Reply:
These values are not meant to be provided by the user. Instead, Luigi chooses one, and puts it in place of `VALUE` to the query params in the URL (e.g. `http://luigi.com/logout.html?reason=tokenExpired&thirdPartyCookies=disabled`. This is then used (or rather may be used), by the application developer, to provide different versions of logout page (one for disabled, one for enabled etc.).

Reply:
done.